app/assets/javascripts/discourse/app/components/composer-container.gjs

```diff
@@ -292,11 +297,6 @@ export default class ComposerContainer extends Component {
                 {{/if}}
 
                 {{#if this.site.mobileView}}
-                  {{#if this.composer.whisperOrUnlistTopic}}
-                    <span class="whisper">
-                      {{icon "far-eye-slash"}}
```

Comment:
This wasn't listed in PR description, but I guess it makes sense to remove from mobile view. It's cleaner, right?

Reply:
Ah yes, makes sense!